translation/models/translation_services.py

```diff
@@ -125,6 +130,14 @@ def get_available_languages(self, pk, obj_type):
             if not lesson:
                 return None
             steps = lesson.first().steps.all()
+        elif obj_type == RequestedObject.LESSON:
+            course = TranslatedLesson.objects.filter(stepik_id=pk, service_name=self.service_name)
+            if not course:
+                return None
+            steps = []
+            for lesson in course.lessons.all():
+                for step in lesson.steps.all():
+                    steps.append(step)
```

Comment:
I think that we can `extend` steps array without second for-loop. (but it doesn't matter)